ReviewSite/scripts/data/example_reviews.js

```diff
@@ -0,0 +1,34 @@
+let example_reviews = [
```

Comment:
can u pls make ur search case insensitive and alike, so if I enter 'the' - The Hobbit will be available displayed.
<img width="519" alt="Screenshot 2022-04-12 at 12 35 12" src="https://user-images.githubusercontent.com/43992068/162942082-23b73046-230e-408d-a45c-2cc8bdcbf166.png">